src/components/App.jsx

```diff
@@ -26,6 +29,11 @@ class App extends React.Component {
   }
 
   render() {
+    if (this.props.projectStatus !== PROJECT_STATUS.READY ||
+        this.props.workflowStatus !== WORKFLOW_STATUS.READY) {
+      return <div>Loading...</div>;
```

Comment:
This makes sense from a performance standpoint, but we should prioritize those simple changes (error handling and styling the loading... in time for beta). Thanks for pointing this out with the Future TODO.